### PR TITLE
Add docs for utilizing purchased line items inside the asset totals a…

### PR DESCRIPTION
**Pull request by daallen303**
…rray

### CODE REVIEW

File: src/content/guides/line-items.mdx [hunk 1]
```diff
@@ -56,3 +56,32 @@ nav:
       }
     ]
     ```
+
+## Determining Paid For Line Items for Partial Payments
```
Comment (IsaacMurtagh) [inline src/content/guides/line-items.mdx:60]: I believe this is independant of partial payments.
Maybe there should just be some context on ways this is useful, like a partial payment.

File: src/content/guides/line-items.mdx [hunk 2]
```diff
@@ -56,3 +56,32 @@ nav:
...
+
+Payments via Centrapay can be made against specific line items. This can happen because a user has specifically requested the line items they wish an asset to pay for, or because they were using an asset that can only be redeemed for a specific product like a Centrapay Token.
+It's important to communicate to the merchant which line items have been purchased via Centrapay so they are able to make educated decisions when refunding or completing partially paid transactions.
+It is also important for reporting to keep track of which assets purchased which line item, this information should be displayed to a consumer on their receipt if possible.
+Line items that were purchased by a specific asset, will be returned on a [Payment Request](https://docs.centrapay.com/api/payment-requests#payment-requests) inside the [Asset Totals](https://docs.centrapay.com/api/payment-requests#asset-total) array.
+In order to display this information to a user integrators must iterate through the assetTotals array and display any line items purchased to the merchant. Line items that have been purchased via Centrapay should no longer be able to be purchased using another form of payment.
```
Comment (IsaacMurtagh) [inline src/content/guides/line-items.mdx:66]:
```suggestion
In order to display this information to a user, integrators must iterate through the `assetTotals` array and display any line items purchased to the merchant. Line items that have been purchased via Centrapay should no longer be able to be purchased using another form of payment.
```

File: src/content/guides/line-items.mdx (hunk 1 above)
Comment (daallen303) [reply src/content/guides/line-items.mdx]: I removed " for Partial Payments" and added a bit more context/reworded the paragraph below. Could you give it another read please? :)

File: src/content/guides/line-items.mdx [hunk 3]
```diff
@@ -56,3 +56,32 @@ nav:
...
+It's important to communicate to the merchant which line items have been purchased via Centrapay so they are able to make educated decisions when refunding transactions or completing partial payments.
+This information should also be displayed to a consumer when possible. If any line items were purchased by a specific asset, they will be returned on a [Payment Request](https://docs.centrapay.com/api/payment-requests#payment-requests) inside the [Asset Totals](https://docs.centrapay.com/api/payment-requests#asset-total) array.
+Integrators must iterate through the `assetTotals` array and display any line items purchased to the merchant. Line items that have been purchased via Centrapay must no longer be able to be purchased using another form of payment.
+This is especially important for any assets which are only redeemable against specific line items. For example, there could be a case where someone used a digital token to pay for a line item, and then a physical voucher to pay for that same line item, which would mean the second voucher could go towards purchase something other than the intended line item.
```
Comment (IsaacMurtagh) [inline src/content/guides/line-items.mdx:66]:
```suggestion
This is especially important for any assets which are only redeemable against specific line items. For example, there could be a case where someone used a digital token to pay for a line item, and then a physical voucher to pay for that same line item, which would mean the second voucher could go towards purchasing something other than the intended line item.
```

**Review by IsaacMurtagh — APPROVED** [review]
Looks good, just one typo

File: src/content/guides/line-items.mdx [hunk 4]
```diff
@@ -56,3 +56,32 @@ nav:
       }
     ]
     ```
+
+## Determining Paid For Line Items for Partial Payments
+
+Payments via Centrapay can be made against specific line items. This can happen because a user has specifically requested the line items they wish an asset to pay for, or because they were using an asset that can only be redeemed for a specific product like a Centrapay Token.
+It's important to communicate to the merchant which line items have been purchased via Centrapay so they are able to make educated decisions when refunding or completing partially paid transactions.
+It is also important for reporting to keep track of which assets purchased which line item, this information should be displayed to a consumer on their receipt if possible.
```
[the remark this answers is not in the document]
Comment (daallen303) [reply src/content/guides/line-items.mdx]: This section has been reworded. :)

**Pull request merged**